routes/authors.js

```diff
@@ -1,7 +1,72 @@
 const express = require("express");
 const generateId = require("uuid/v4");
-
 const router = express.Router();
+const authors = require("../data/authors");
+const books = require ("../data/books");
+
+
+
+router.get('/', (req, res) => {
+  res.json(authors);
+});
+router.post('/', (req, res) => {
+  const id = generateId();
+  const {
+    name,
+    website
+  } = req.body;
+  authors.push({
+    id,
+    name,
+    website
+  })
+  res.json(authors);
+});
+
+router.get('/:id', (req, res) => {
+  const id = req.params.id
+  const authorSearched = authors.filter(obj => {
+    return obj.id == id
+  });
+  res.json(authorSearched);
```

Comment:
You want to send only one author, but `filter` returns an array.
You would rather use the `find` method.